### DISCUSSION TITLE
Wrong Island/Overhang detection?

### DISCUSSION THREAD

**Discussion by srozinov** [sn4k3/UVtools#562] [Q&A]
Hi! 

Used UVTools v3.6.6 to check a model.
Can you explain why layer 80 classified Island and Overhang in the following model?
BTW it has been printed well with no defects. I supposed there are enough supports for the layer 80,
and ignored warning of UVTools.
Could you please advise?

https://drive.google.com/file/d/1FQVtf8q3z_QIdi_WlI3yxenOjshP7I80/view?usp=sharing

**Comment by sn4k3** [comment]
Because it does not meet your requirements, supports below are considered as unsafe / too few to suport that mass, you can adjust the threshold or ignore this one.

![image](https://user-images.githubusercontent.com/113281/192613262-b1ea349d-87b7-4dd9-8397-6e46e09a798f.png)

About overhang i have no idea, worse when it show two and one is repeated i will check
